symplyphysics/laws/dynamics/vector/force_is_derivative_of_momentum.py

```diff
@@ -0,0 +1,53 @@
+from sympy import Derivative, sympify
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    Vector,
+    validate_input,
+    validate_output,
+    scale_vector,
+    add_cartesian_vectors,
+)
+from symplyphysics.core.vectors.vectors import QuantityVector
+
+# Description
+## Newton's second law of motion can be generalized in terms of the vector of linear momentum.
+
+# Law:
+## F = dp/dt
+## F - force vector
+## p - vector of linear momentum
+
+time = Symbol("time", units.time)
+
+
+def force_law(momentum_: Vector) -> Vector:
+    force_components = list(map(
+        lambda component: Derivative(component, time),
+        momentum_.components,
+    ))
+    return Vector(force_components, momentum_.coordinate_system)
+
+
+@validate_input(
+    momentum_before_=units.mass * units.velocity,
+    momentum_after_=units.mass * units.velocity,
+    time_=time,
+)
+@validate_output(units.force)
+def calculate_force(
+    momentum_before_: QuantityVector,
+    momentum_after_: QuantityVector,
+    time_: Quantity,
+) -> QuantityVector:
+    # delta_p = (t / delta_t) * (p1 - p0)
+    momentum_function = scale_vector(
```

Comment:
Use quantity version of vector arithmetics.

Reply:
If I replace them with their quantity versions, `scale_quantity_vector` throws an error `symplyphysics.core.errors.UnitsError: Argument 'QTY38[0]' to function 'QuantityVector' should not contain free symbols`. It works with non-quantity functions, though.

Reply:
I will look into it.